athena-federation-sdk/src/main/java/com/amazonaws/athena/connector/lambda/handlers/GlueMetadataHandler.java

```diff
@@ -114,6 +114,8 @@
     public static final String DATETIME_FORMAT_MAPPING_PROPERTY = "datetimeFormatMapping";
     // Table property (optional) that we will create from DATETIME_FORMAT_MAPPING_PROPERTY with normalized column names
     public static final String DATETIME_FORMAT_MAPPING_PROPERTY_NORMALIZED = "datetimeFormatMappingNormalized";
+    // Table property (optional) that we expect to contain any types to be overridden for a column, ex) timestamp with tz
+    public static final String TYPE_OVERRIDE_MAPPING_PROPERTY = "typeOverrideMapping";
```

Comment:
+1 we need clear documenation on this. I'm also unclear if we really need this since Glue doesn't validate what you put into the type field. its just a string i believe.